include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -259,23 +259,22 @@ ERROR(failed_base_method_call_synthesis,none,
       (ValueDecl *, ValueDecl *))
 
 ERROR(both_returns_retained_returns_unretained, none,
-      "%0 cannot be annotated with both swift_attr('returns_retained') and "
-      "swift_attr('returns_unretained') attributes",
+      "%0 cannot be annotated with both 'SWIFT_RETURNS_RETAINED' and "
+      "'SWIFT_RETURNS_UNRETAINED'",
```

Comment:
I think I had suggested this nit elsewhere, but since you're editing these diagnostic messages already, it might be a good opportunity to remove the apostrophes from the diagnostic messages:

```suggestion
      "%0 cannot be annotated with both SWIFT_RETURNS_RETAINED and "
      "SWIFT_RETURNS_UNRETAINED",
```

Justification: these are library-provided "keywords" rather than user-defined identifiers.

But if there are a ton of other diagnostics that follow the opposite convention (i.e., use apostrophes for library-defined annotations), then keep it as is.

Reply:
@egorzhdan do you have thoughts about this? I think we should change the diagnostic messages in ClangImporter as @j-hui mentioned to make sure that apostrophes are used only for user-defined identifiers.